pundit-matchers.gemspec

```diff
@@ -12,8 +12,7 @@ Gem::Specification.new do |s|
   s.require_paths = ['lib']
   s.homepage    = 'https://github.com/punditcommunity/pundit-matchers'
   s.license     = 'MIT'
-  s.add_runtime_dependency 'rspec-rails', '>= 3.0.0'
-  s.add_development_dependency 'pundit', '~> 1.1', '>= 1.1.0'
```

Comment:
also, this is ignored when using the gem in a third-party and pundit is not used to test matchers. I've dropped it